pivotal-tracker/pivotal_integration.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+// Get the environment; we will post a new comment to Tracker each time a commit appears on Pantheon.
+$env = $_ENV['PANTHEON_ENVIRONMENT'];
+
+// Do not watch test or live, though.
```

Comment:
Maybe the script should behave differently when processing commits in multidev environments, and avoid setting the issue status until the commit goes to the `dev` branch (but still post comments on commits on commits in multidev branches).